include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 3
+#define SOF_ABI_MINOR 4
```

Comment:
is this really an ABI change if you just rename the fields?

Reply:
Technically it's not, so it really depends what is our approach here. As Liam already stated in https://github.com/thesofproject/sof/pull/1131 there is no harm with increment. I can do it both ways, so your decision.

Reply:
Probably best then not to bump ABI. I'll merge FW part once ABI removed.

Reply:
@lgirdwood @plbossart Both PRs updated.